### ISSUE TITLE
Protobuf issue

### ISSUE THREAD

**Issue by 5A11** [fetchai/cosmpy#207]

**Comment by solarw** [comment]
@5A11 Can you add a bit more details on it?

**Comment by solarw** [comment]
@5A11 Can you add a bit more details on it?

**Comment by 5A11** [comment]
Found this relevant report in our internal comms:

```
Hey team, I was trying to use CosmPy on an osmosis node this morning and I ran into a couple of issues:
1. Default installation still gives me protobuf 4.21, which I need to override with pipenv install protobuf==3.20
2. ...
```

**Comment by 5A11** [comment]
Investigate if PR #187 addresses this issue.

**Comment by solarw** [comment]
`PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python` helps to avoid double filre register issue on tests side

but during aea test run see error with `TypeError: Parameter to MergeFrom() must be instance of same class: expected cosmos.base.v1beta1.coin_pb2.Coin got cosmos.base.v1beta1.coin_pb2.Coin.`
  tells there was double import of Coin class.
  
  switching off docstring unit tests helps to test suit run well. so the problem can be caused by pytest tests resolution and importing

**Comment by solarw** [comment]
the issue was caused by outdated tools used to generate cosmos sdk proto files.
protoc version was 3.13
after update of grpcio-tools (used for generation) and regenerating proto files, issue has gone away.

so problem was: mismatch of generator and python protobuf library

**Comment by 5A11** [comment]
This is now fixed in #215
